Userland/Applications/Calendar/AddEventDialog.cpp

```diff
@@ -53,8 +53,8 @@ AddEventDialog::AddEventDialog(Core::DateTime date_time, EventManager& event_man
     auto widget = set_main_widget<GUI::Widget>();
     widget->load_from_gml(add_event_dialog_gml).release_value_but_fixme_should_propagate_errors();
 
-    auto event_title_textbox = widget->find_descendant_of_type_named<GUI::TextBox>("event_title_textbox");
-    event_title_textbox->set_focus(true);
+    auto& event_title_textbox = *widget->find_descendant_of_type_named<GUI::TextBox>("event_title_textbox");
+    event_title_textbox.set_focus(true);
```

Comment:
These convert-pointer-to-reference changes aren't really related to this commit, and would be better separate.